lib/myelements/tools_poly.py

```diff
@@ -343,5 +361,4 @@ def convex_hull(points):
                 pt2 = hull[-2]
                 l = is_left(pt2, pt1, p)
             hull.append(p)
-    return hull     
-    
```

Comment:
Hey! I haven't checked for flake8 warnings yet! After this https://github.com/sugarlabs/Bridge/pull/17#issuecomment-613239888 , I'm kind of confused should I make flake8 fixes or leave it for later.

Reply:
Your _changes_ should comply with flake8 where practical.  Please don't make changes for flake8 to source code you haven't yet changed, as that clouds the review.

Reply:
No, its not about flake8 changes, in the diff, I could see that you deleted line 347, while it was already there

Reply:
@JuiP I think we should apply changes and keep in mind flake8 rules. In this may we wont need to fix flake8 warnings again and it will save some time of us.